### ISSUE TITLE
Can't get things to center

### ISSUE THREAD

**Issue by spamalope01** [codefellows/seattle-201n3#36]
## When submitting a request for lab help, please include the following information:

1. What assignment are you working on?
N11/N12 

2. What is the unexpected outcome of your code?

I can't get the logo and verbiage around the logo image to center on the header.  margin: 0 auto is not working.  The closest I can get it is still severely off centered, and i'm not even sure why it displays that close to center...I'm not sure what i'm doing.  

also, when i set the document to 960px, it was the smallest site ever.  i had to set it to 100% to get it to display properly.  

HTML:
```<header>
      <!-- <div id="centeredLogo"> -->
        <h1 id="headTitle">Pat's Best Salmon Cookies</h1>
        <img src="images/salmonMain.png" alt="" id="mainFishImg"/>
        <h3 id='subHead'>Salmon cookies made with fish, salmon cookies are delish!</h3>
      <!-- </div> -->```  (NOTE:  this isn't the only thing in the header...that's why you don't see the closing header tag).

CSS:
```html {
  width: 100%;
}

#headTitle {
  margin: 0 auto;
  width: 50%;
}

#mainFishImg {
  width: 300px;
  margin: 0 auto;
}

#subHead {
  margin: 0 auto;
  width: 50%;
}

3. What is the expected outcome of your code?
The damn things should be centered.

**Comment by sjschmidt44** [comment]
Hey Jonathan. I can help resolve this but I need to take a closer look at your whole code base. If you're in a few minutes early tonight I can help you out then!

**Comment by spamalope01** [comment]
I'll absolutely be in. I may have a fix, but i'm not sure if it's one you're going to be approving of, since it involves divs and text-align: center.  i haven't implemented that fix, but based on some online research, that sounds like it'll work.  

otherwise i'm sure it's something stupid (it always is) like positioning not being declared or i didn't assign some element a width...
 

    On Wednesday, May 11, 2016 2:31 PM, Scott Schmidt <notifications@github.com> wrote:
 

 Closed #36.—
